checkov/common/bridgecrew/integration_features/features/repo_config_integration.py

```diff
@@ -0,0 +1,83 @@
+import logging
+import requests
+import re
+
+from checkov.common.bridgecrew.integration_features.base_integration_feature import BaseIntegrationFeature
+from checkov.common.bridgecrew.platform_integration import bc_integration, BcPlatformIntegration
+from checkov.common.bridgecrew.severities import Severities
+from checkov.common.util.data_structures_utils import merge_dicts
+from checkov.common.util.http_utils import get_default_get_headers, get_auth_header, extract_error_message
+
+# service-provider::service-name::data-type-name
+CFN_RESOURCE_TYPE_IDENTIFIER = re.compile(r"^[a-zA-Z0-9]+::[a-zA-Z0-9]+::[a-zA-Z0-9]+$")
+
+
+class RepoConfigIntegration(BaseIntegrationFeature):
+    def __init__(self, integration_instance: BcPlatformIntegration = bc_integration):
+        super().__init__(integration_instance, order=0)
+        self.skip_paths = []
+        self.code_review_threshold = None
+        self.code_review_skip_policies = []
```

Comment:
```suggestion
        self.code_review_skip_policies = set()
```
change this to a `set`